crates/header-translator/src/method.rs

```diff
@@ -535,28 +535,29 @@ impl Method {
             .get_objc_type_encoding()
             .expect("method to have encoding");
 
-        Some((
-            modifiers.designated_initializer,
-            Method {
-                selector,
-                fn_name,
-                availability,
-                is_class,
-                is_optional: entity.is_objc_optional(),
-                memory_management,
-                arguments,
-                result_type,
-                is_error,
-                safe: data.unsafe_.safe(),
-                is_pub,
-                non_isolated: modifiers.non_isolated,
-                mainthreadonly,
-                weak_property: false,
-                must_use: modifiers.must_use,
-                encoding,
-                documentation: Documentation::from_entity(&entity, context),
-            },
-        ))
+        let mut method = Method {
+            selector,
+            fn_name,
+            availability,
+            is_class,
+            is_optional: entity.is_objc_optional(),
+            memory_management,
+            arguments,
+            result_type,
+            is_error,
+            safe: data.unsafe_.safe(),
+            is_pub,
+            non_isolated: modifiers.non_isolated,
+            mainthreadonly,
+            weak_property: false,
+            must_use: modifiers.must_use,
+            encoding,
+            documentation: Documentation::from_entity(&entity, context),
+        };
+
+        method.apply_config_overrides(&data);
```

Comment:
I'd like this to happen _before_ construction of `Method`, and I'd like it to also happen when constructing properties. Maybe make `apply_config_overrides` a free-standing function instead?